partiql-logical-planner/src/lower.rs

```diff
@@ -34,6 +36,7 @@ type FnvIndexMap<K, V> = IndexMap<K, V, FnvBuildHasher>;
 enum QueryContext {
     FromLet,
     Path,
+    Order,
```

Comment:
I see `QueryContext::Order` is only used in `enter_sort` but not used anywhere else. Is this intentional? Or part of some upcoming data flow analysis work?